samples/DocumentTaskExample/Program.cs

```diff
@@ -0,0 +1,209 @@
+            {
+                UserId = userId;
+                UserName = userName;
+                DirectoryProvider = directoryProvider;
+                Mention = mention;
+                Email = email;
+            }
+
+            public string UserId { get => _userId; set => _userId = value; }
+
+            public string UserName { get => _userName; set => _userName = value; }
+
+            public string DirectoryProvider { get => _directoryProvider; set => _directoryProvider = value; }
+
+            public string Mention { get => _mention; set => _mention = value; }
+
+            public string Email { get => _email; set => _email = value; }
+        }
+
+        private static void Main(string[] args)
+        {
+            // Must have one or two arguments.
+            if (args.Length < 0)
+            {
+                ShowHelp();
+                return;
+            }
+
+            if (!File.Exists(args[0]))
+            {
+                Console.WriteLine("File {0} does not exist.", args[0]);
+                ShowHelp();
+                return;
+            }
+
+            Console.WriteLine("Processing presentation: {0}", args[0]);
+
+            using (WordprocessingDocument docPackage = WordprocessingDocument.Open(args[0], true))
+            {
+                StringBuilder newFn = new StringBuilder(args[0]);
+                newFn.Replace(".docx", "_new.docx");
+
+                WordprocessingDocument newDocumentPackage = (WordprocessingDocument)docPackage.SaveAs(newFn.ToString());
+                newDocumentPackage.AddParagraphIdFeature();
+                MainDocumentPart mdp = newDocumentPackage.MainDocumentPart;
+
+                string strCommentId = "3";
+                string comment = " Here's another sentence that is just too long. Shorten it please.";
+                string run = "The introduction to this article.";
+
+                // userids, providers and names will have to be programmatically accessed via directory services and must be valid. These are just for example
+                User tony = new("S::john.doe@contoso.com::3063813b-f01d-4030-9808-501a178e7963", "John Doe", "AD", "@John Doe", "john.doe@contoso.com");
+                User bruce = new("S::jane.doe@contoso.com::ec6240b1-52a3-46dd-9697-ef7bcc7a29e8", "Jane Doe", "AD", "@Jane Doe", "jane.doe@contoso.com");
+
+                AddNewParagraphRunWithComment(mdp, strCommentId, run);
+
+                AddMentionComment(mdp, strCommentId, bruce.Mention, comment, tony, bruce);
+
+                StringBuilder taskStr = new StringBuilder(bruce.Mention + " " + comment);
+                AddNewTask(mdp, taskStr.ToString(), bruce, tony);
+
+                newDocumentPackage.Save();
+                newDocumentPackage.Close();
+                docPackage.Close();
+            }
+        }
+
+        private static void ShowHelp()
+        {
+            Console.WriteLine("doctaskssample: ");
```

Comment:
I wonder if we should have a single entry point for all the samples that can dynamically grow to adjust for each sample

Reply:
No need to block this PR on that though

Reply:
It already exists, I need to add it to this file